packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -81,6 +82,30 @@ const menuStyles = css`
   z-index: 5;
 `;
 function FilterMenu(props: TFilterMenuProps) {
+  const [headerSelectOptions, setHeaderSelectOptions] = useState<string>();
+  const renderOperatorsInput = (): ReactNode => {
+    const operatorOptions = [
+      { value: 'is', label: 'is' },
+      { value: 'is not', label: 'is NOT' },
+    ];
+
+    return (
+      <SelectInput
+        appearance="quiet"
+        isCondensed={true}
+        isSearchable={false}
+        value={
+          // Default to the first option if no value is passed
+          headerSelectOptions ? headerSelectOptions : operatorOptions[0].value
+        }
+        options={operatorOptions}
+        onChange={(event) => {
+          setHeaderSelectOptions(event.target.value as string);
+        }}
+      />
+    );
+  };
```

Comment:
I saw the comment below, but what is speaking against moving this part into the story?

Reply:
Totally agree! updated here: https://github.com/commercetools/ui-kit/pull/2951/commits/87f00d071d260cc8ba05dc38380993e55887aeca